modules/registry-configuring-storage-vsphere.adoc

```diff
@@ -10,35 +10,31 @@
 [id="registry-configuring-storage-vsphere_{context}"]
 = Configuring registry storage for VMware vSphere
 
-As a cluster administrator, following installation you must configure your
-registry to use storage.
+As a cluster administrator, following installation you must configure your registry to use storage.
 
 .Prerequisites
 
 * Cluster administrator permissions.
 * A cluster on VMware vSphere.
-* Provision persistent storage for your cluster. To deploy a private image registry, your storage must provide
-ReadWriteMany access mode.
+* Persistent storage provisioned for your cluster, such as Red Hat OpenShift Container Storage.
 +
 [IMPORTANT]
 ====
-vSphere volumes do not support the `ReadWriteMany` access mode. You must use
-a different storage backend, such as object storage, to configure the registry storage for high-availability.
+{product-title} supports `ReadWriteOnce` access for image registry storage when you have only one replica. To deploy a private image registry, or to configure registry storage for high availability with two or more replicas, you must use a different storage back end that supports `ReadWriteMany`, such as object storage.
```

Comment:
Same here, please remove "such as object storage"; why to mention "To deploy a private image registry"?

Reply:
OK, removed "such as object storage" in both modules.

The "private image registry" was wording left in from a previous iteration. Not sure the history behind it, but I agree with your suggestion of instead using "an image registry that supports high availability", which is more clear to me than "private image registry".